scripts/fast-reboot-dump.py

```diff
@@ -49,24 +53,56 @@ def get_vlan_ifaces():
 
     return vlans
 
+def get_bridge_port_id_2_port_id(db):
+    bridge_port_id_2_port_id = {}
+    keys = db.keys(db.ASIC_DB, 'ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT:oid:*')
```

Comment:
is it possible the asic db is not completely initialized and you do not have all bridge_port id to port id mappings?